astropy/modeling/physical_models.py

```diff
@@ -317,3 +319,69 @@ def bounding_box(self, factor=50):
         dx = factor * self.fwhm
 
         return (x0 - dx, x0 + dx)
+
+
+class Plummer1D(Fittable1DModel):
+    r"""One dimensional Plummer density profile model.
+
+    Parameters
+    ----------
+    mass : float
+        Total mass of cluster.
+    r_plum : float
+        Scale parameter which sets the size of the cluster core.
+
+    Notes
+    -----
+    Model formula:
+
+    .. math::
+
+        \rho(r)=\frac{3M}{4\pi a^3}(1+\frac{r^2}{a^2})^{-5/2}
+
+    References
+    ----------
+    .. [1] http://adsabs.harvard.edu/full/1911MNRAS..71..460P
+    """
+
+    mass = Parameter(default=1.0)
+    r_plum = Parameter(default=1.0)
+
+    @staticmethod
+    def evaluate(x, mass, r_plum):
+        """
+        Evaluate plummer density profile model.
+        """
+        return (3*mass)/(4 * np.pi * r_plum**3) * (1+(x/r_plum)**2)**(-5/2)
+
+    @staticmethod
+    def fit_deriv(x, mass, r_plum):
+        """
+        Plummer1D model derivatives.
+        """
+        d_mass = ((6 * x**2 * mass) - (9*r_plum**2 * mass)) / \
+                 (4*np.pi * np.sqrt(1 + (x/r_plum)**2) *
+                  ((x**2 + r_plum**2)**3)) + 12
+        d_r_plum = 3 / (4*np.pi * r_plum**3 * (1 + (x/r_plum)**2)**(5/2))
+
```

Comment:
The test is failing because there's a problem with the derivative.  The values of `d_mass` and `d_r_plum` are reversed. There might be a problem with `d_r_plum` as well. Using online calculator I get

`d_r_plum = (6*m*x**2 - 9*m*r**2) / (4*np.pi *r**6 * (1+x**2/r**2)**7/2`

Could you double check the derivative?

Reply:
whoops you're right! I checked it online and it should be correct now.